src/include/btmem.h

```diff
@@ -633,6 +633,8 @@ struct __wt_split_page_hist {
  */
 struct __wt_page_block_meta {
     uint64_t page_id;
+    uint64_t checkpoint_id;
+    uint64_t reconciliation_id;
 
     /* TODO: Place additional metadata here, e.g., checkpoint ID, number of deltas. */
```

Comment:
Should we, at this point, add `backlink_checkpoint_id` and `base_checkpoint_id`? We would need to get these in `plh_get` and then populate them here.

If we are already adding fields that we would need later so that we would not change the interface, we would also need to add `num_deltas` (since the last full image) here and to be populated in `plh_get`.

Reply:
These are now added.